packages/manager/modules/error-page/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@ovh-ux/manager-error-page",
+  "version": "0.0.0",
+  "private": true,
+  "homepage": "https://github.com/ovh-ux/manager/tree/master/packages/manager/modules/error-page#readme",
```

Comment:
`homepage`, `bugs` and `repository` field could be dropped for a `private` package I guess. Isn't it?